fluentui_core/src/main/java/com/microsoft/fluentui/compose/ModalPopup.kt

```diff
@@ -56,53 +67,70 @@ import java.util.UUID
  */
 @Composable
 fun ModalPopup(
-    onDismissRequest: () -> Unit,
+    onDismissRequest:(() -> Unit)? = null,
```

Comment:
why are we making this optional?

Reply:
It is no longer required, the default back press behavior will navigate back to the parent activity now

Reply:
just to understand this more, this onDismissRequestCallback is for any action required to be perfomed on dismiss right?
Like even if it's navigating back to the parent activity, there could be some action required to be perfomed on dismiss?

Reply:
Yes, but the default behavior for any view is to close on back button press now.
So if we don't explicitly give a On Dismiss function, this will still work.